interfaces/cython/cantera/transport.pyx

```diff
@@ -144,7 +144,6 @@ cdef class Transport(_SolutionBase):
                     model = 'None'
                 self.transport = newTransportMgr(stringify(model), self.thermo)
             self._transport.reset(self.transport)
-
         super().__init__(*args, **kwargs)
```

Comment:
I am trying. It is difficult to find. Do you have any suggested methods?

Reply:
If you run `git log --patch interfaces/cython/cantera/transport.pyx`, it will only show you changes to this file, which should help you find the commit that introduced the change. I think in this case, this is a result of a change which was later mostly reverted.